homeassistant/components/zwave_js/trigger.py

```diff
@@ -2,45 +2,17 @@
 
 from __future__ import annotations
 
-from homeassistant.const import CONF_PLATFORM
-from homeassistant.core import CALLBACK_TYPE, HomeAssistant
-from homeassistant.helpers.trigger import (
-    TriggerActionType,
-    TriggerInfo,
-    TriggerProtocol,
-)
-from homeassistant.helpers.typing import ConfigType
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.trigger import Trigger
 
 from .triggers import event, value_updated
 
 TRIGGERS = {
```

Comment:
```py
from homeassistant.util.decorator import Registry

TRIGGERS = Registry()
```

Reply:
I don't think this kind of global variable is a pattern we should encourage

Reply:
This ended up not working.